modules/understanding-machine-config-operator.adoc

```diff
@@ -41,5 +41,8 @@ KubeletConfig custom resource (CR).
 
 [IMPORTANT]
 ====
-To prevent control plane nodes from autorebooting after machine config changes are applied, you must pause the autoreboot process by setting the `spec.paused` field to `true` in the machine pool config.
+When changes are made to a machine configuration, the Machine Config Operator automatically reboots all corresponding nodes in order for the changes to take effect.
+
+To prevent the nodes from automatically rebooting after machine configuration changes, before making the changes you must pause the autoreboot process by setting the `spec.paused` field to `true` in the corresponding machine config pool. When paused, machine configuration changes are not applied until you set the `spec.paused` field to `false` and the nodes have rebooted into the new configuration.
```

Comment:
```suggestion
To prevent the nodes from automatically rebooting after machine configuration changes, before making the changes, you must pause the autoreboot process by setting the `spec.paused` field to `true` in the corresponding machine config pool. When paused, machine configuration changes are not applied until you set the `spec.paused` field to `false` and the nodes have rebooted into the new configuration.
```

Reply:
@kalexand-rh Thanks. I wasn't sure about that comma. Doesn't seem parenthetical.